### PR TITLE
Add jQuery.BgSwitcher@0.4.3 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10646

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): http://github.com/rewish/jquery-bgswitcher
 * License and its reference: [MIT](https://github.com/rewish/jquery-bgswitcher/blob/master/LICENSE.md)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 15
   - Count of stars: 141
   - Count of forks: 55

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@extend1994 you forgot to update your master branch?

**Comment by extend1994** [comment]
@PeterDaveHello I forgot to rebase the branch, sorry.
@hare1039 I don't take the url you mentioned as the homepage because it's only a demo page and without documentation, download information...etc. They are requirements of a homepage.

**Comment by extend1994** [comment]
@clairetsai818 @sufuf3 I didn't add them because they are just beta versions. Please help me review it again, if there is no other problem, then I will rebase the branch, thanks.

File: ajax/libs/jQuery.BgSwitcher/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jQuery.BgSwitcher",
+  "filename": "jquery.bgswitcher.min.js",
+  "version": "0.4.3",
+  "description": "Switch the background image with using effect.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rewish/jquery-bgswitcher.git"
+  },
+  "author": "rewish",
```
Comment (pvnr0082t) [inline ajax/libs/jQuery.BgSwitcher/package.json]: @extend1994 We can use `author` info as the same as upstream. ref: https://github.com/rewish/jquery-bgswitcher/blob/master/bgswitcher.jquery.json#L5-L9.

**Comment by extend1994** [comment]
@pvnr0082t I modified the author info, please help me review it, thank you.

File: ajax/libs/jQuery.BgSwitcher/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/jQuery.BgSwitcher/package.json]: @cdnjs/intern3 a single string sometimes is meaningless and I don't want to you add the single string by yourselves.

**Comment by PeterDaveHello** [comment]
If it at least links to its GitHub repo with a demo, I'll treat it as homepage @cdnjs/intern3

**Comment by PeterDaveHello** [comment]
@extend1994 please send a pull request for its homepage.

**Pull request merged**